### ISSUE TITLE
lib/overload.t: blocker to strict-by-default

### ISSUE THREAD

**Issue by jkeenan** [atoomic/perl#229]
As of commit ac6b7ccf73 in the alpha-dev-02-strict branch on Aug 18 2020, the file cited is failing.  Here is output from `cd t; ./perl harness -v <file>; cd -`:
```
# ../lib/overload.t

Global symbol "$c" requires explicit package name (did you forget to declare "my $c"?) at ../lib/overload.t line 64.
Global symbol "$c" requires explicit package name (did you forget to declare "my $c"?) at ../lib/overload.t line 66.
Global symbol "$c" requires explicit package name (did you forget to declare "my $c"?) at ../lib/overload.t line 67.
Global symbol "$c" requires explicit package name (did you forget to declare "my $c"?) at ../lib/overload.t line 68.
Global symbol "$c" requires explicit package name (did you forget to declare "my $c"?) at ../lib/overload.t line 81.
Global symbol "$c" requires explicit package name (did you forget to declare "my $c"?) at ../lib/overload.t line 82.
Global symbol "$c" requires explicit package name (did you forget to declare "my $c"?) at ../lib/overload.t line 84.
Global symbol "$c" requires explicit package name (did you forget to declare "my $c"?) at ../lib/overload.t line 86.
Global symbol "$dummy" requires explicit package name (did you forget to declare "my $dummy"?) at ../lib/overload.t line 111.
Global symbol "$dummy" requires explicit package name (did you forget to declare "my $dummy"?) at ../lib/overload.t line 111.
...
```
This is a case where an attempt at properly scoping those variables led down a rabbit hole.  Need fresh eyes.

@atoomic @toddr 

Thank you very much.
Jim Keenan

**Comment by atoomic** [comment]
this is now fixed by eea74faa

**Comment by jkeenan** [comment]
> this is now fixed by [eea74fa](https://github.com/atoomic/perl/commit/eea74faab175d786090bf6407a57fa7717431898)

Confirmed.
```
$ extract-sha
627eedcc50
$ cd t; TEST_JOBS=1 ./perl harness ../lib/overload.t ../lib/feature.t; cd -
../lib/overload.t .. ok         
../lib/feature.t ... ok     
All tests successful.
Files=2, Tests=5428,  1 wallclock secs ( 0.19 usr  0.00 sys +  0.57 cusr  0.09 csys =  0.85 CPU)
Result: PASS
```
Closing.

Thank you very much.
Jim Keenan
